### PR TITLE
Added sublime_lib dependency.

### PR DESCRIPTION
**Pull request by Thom1729**
Will be used for #155, #67, and probably many other things.

I'm not familiar with PowerShell; we'll need to add sublime_lib in there as well. I also have not tested the CI integration.

Ideally, we could move most of the installation code into Python, relying on `pathlib` and other built-in cross-platform functionality. In a perfect world, it would be nice to make this a full-fledged pypi package.

### CODE REVIEW

**Comment by Thom1729** [comment]
@randy3k Any chance you can take a look at the PowerShell stuff? I've never worked with it and I don't have a Windows machine to test it with.

File: sbin/circleci.sh [hunk 1]
```diff
@@ -68,6 +68,24 @@ Bootstrap() {
         echo
     fi
 
+    SUBLIME_LIB_PATH="$STP/coverage"
```
Comment (randy3k) [inline sbin/circleci.sh]: This path should `$STP/sublime_lib`

File: sbin/travis.sh [hunk 2]
```diff
@@ -73,6 +73,24 @@ Bootstrap() {
         echo
     fi
 
+    SUBLIME_LIB_PATH="$STP/coverage"
```
Comment (randy3k) [inline sbin/travis.sh]: same here

**Comment by deathaxe** [comment]
If a package requests a library, it is installed by Package Control.

**Pull request closed without merging**